### PR TITLE
Adds and updates descriptions on a ton of things

### PR DESCRIPTION
**Pull request by NightmareChamillian**
## About the PR 
This rewords a lot of descriptions, changes a lot more, and adds a few to the more common things that straight up needed them. Notable examples are flashes, manufactureres, the nanofabricator, hyposprays, the chemaster 3k, syringes, reagent extractor, and sec and med vendors

## Why's this needed?
90% of the time, when someone sees something they don't recognize, they'll examine it to try and figure out how it works. Adding clues as how to use something and just better explaining the purpose of things makes it a lot easier to get accustomed to the game.

### CODE REVIEW

File: code/obj/machinery/manufacturer.dm [hunk 1]
```diff
@@ -2518,14 +2518,17 @@
...
+	desc = "A 3D printer-like machine that can construct a variety of items. This one is for producing items useful in surviving extreme scenarios."
 	icon_state = "fab-crates"
 	icon_base = "crates"
 	free_resource_amt = 50
 	free_resources = list(/obj/item/material_piece/steel,
 		/obj/item/material_piece/copper,
 		/obj/item/material_piece/glass,
-		/obj/item/material_piece/cloth/cottonfabric)
+		/obj/item/material_piece/cloth/cottonfabric
+		/obj/item/material_piece/bohrum,// adding these just so it can print everything from the get-go
```
Comment (Zonespace27) [inline code/obj/machinery/manufacturer.dm:2529]: atomize this out of the PR, completely unrelated change

File: code/obj/machinery/manufacturer.dm [hunk 2]
```diff
@@ -2518,14 +2518,17 @@
...
+		/obj/item/material_piece/uqill
```
Comment (Zonespace27) [inline code/obj/machinery/manufacturer.dm:2530]:
```suggestion
		/obj/item/material_piece/uqill,
```

File: code/obj/machinery/manufacturer.dm [hunk 3]
```diff
@@ -2518,14 +2518,17 @@
...
+		/obj/item/material_piece/claretine,)
```
Comment (Zonespace27) [inline code/obj/machinery/manufacturer.dm:2531]:
```suggestion
		/obj/item/material_piece/claretine)
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/vending/vending.dm [hunk 4]
```diff
@@ -2247,6 +2247,7 @@
 		product_list += new/datum/data/vending_product(/obj/item/device/igniter, 8)
 		product_list += new/datum/data/vending_product(/obj/item/device/radio/signaler, 8)
 		product_list += new/datum/data/vending_product(/obj/item/wirecutters, 1)
+		product_list += new/datum/data/vending_product(/obj/item/screwdriver, 1)
```
Comment (Zonespace27) [inline code/modules/vending/vending.dm:2250]: atomization is suggested here as well

**Review by ZeWaka — APPROVED** [review]

**Comment by ZeWaka** [comment]
unnecessary changelog. read the contributor guidelines
